### ISSUE TITLE
Option to Show only project titles in Starred, instead of including the developer's username first

### ISSUE THREAD

**Issue by aha999** [thermatk/FastHub#104]
In the Starred menu, it's annoying that you first have to go read a developer's usernames in order to read/find the project you're looking for. It would be way better if only the project titles were shown, or the option was there to set/toggle it.

Example: 

**Currently:** 
- _UriahShaulMandel/Baldphone_ 

**With the wanted feature/toggle it could look like:** 
- _Baldphone_      

Or, the repo/username text could in smaller letters above the project

**Comment by JohnVeness** [comment]
Yes, I often have this problem of knowing a project name but not knowing the owner and having to scroll to find it.

One possible solution would be to add sorting options, so we could sort by repo name (using author as secondary sort in the event of a clash).
